recipes/selectolax/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "selectolax" %}
+{% set version = "0.3.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/selectolax-{{ version }}.tar.gz
+  sha256: d71d7ae7bec6e856b83111f270bc783e531d86e02ad3a78a37d8591ff8f3620f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - pip
+    - python
+  run:
+    - python
+    - cython >=0.29.23    
```

Comment:
I don't think cython is a run requirement. It's mentioned as a `setup_requires` entry in `setup.py`, so that means is a `host` dependency (with this version constrain). Move this line to `host` and remove the bare `cython` there.

Reply:
@jaimergp I tried what you suggested. Looks like cython is a requirement (if pip-check is necessary).

```sh
import: 'selectolax'
+ pip check
selectolax 0.3.6 requires cython, which is not installed.
Tests failed for selectolax-0.3.6-py37h5e8e339_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
WARNING:conda_build.build:Tests failed for selectolax-0.3.6-py37h5e8e339_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
WARNING conda_build.build:tests_failed(2970): Tests failed for selectolax-0.3.6-py37h5e8e339_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
TESTS FAILED: selectolax-0.3.6-py37h5e8e339_0.tar.bz2
```